### DISCUSSION TITLE
Branching strategy

### DISCUSSION THREAD

**Discussion by Youcef1810** [chocoteam/choco-solver#1108] [General]
Hello everyone,

In the context of allocating takeoff slots to avoid in-flight conflicts, I have developed a CSP model and used the Choco solver. However, the constraint programming in this case is highly complex, and I need a very efficient branching strategy, as the resolution is quite heavy due to the model's complexity and constraints. I need your help, please, as some people have recommended LDS and LNS as branching strategies. I have initial solutions, but I am struggling to implement and adapt them to my code.

Thank you very much for your assistance.

Best regards.

**Comment by cprudhom** [comment]
See #1109
